pkg/pipelines/bootstrap.go

```diff
@@ -46,6 +46,20 @@ type BootstrapOptions struct {
 
 // Bootstrap bootstraps a GitOps pipelines and repository structure.
 func Bootstrap(o *BootstrapOptions, appFs afero.Fs) error {
+	if o.GitOpsWebhookSecret == "" {
+		gitopsSecret, err := secrets.GenerateSecureString(20)
```

Comment:
You probably should make a constant for the len (20).   Also, I would rename the function to secrets.GenerateString()

Reply:
Changes made to incorporate constant and name of function changed.